src/plugins/visualizations/public/wizard/new_vis_modal.test.tsx

```diff
@@ -107,6 +118,30 @@ describe('NewVisModal', () => {
     expect(wrapper.find('[data-test-subj="visType-vis"]').exists()).toBe(true);
   });
 
+  it('should sort promoted visualizations first', () => {
```

Comment:
:information_source: I've added this test mainly, because during work on this I broke it, and the only thing that catched the broken ordering was a snapshot test, where it was not quit clear in the beginning that it was the ordering that was broken. So I added a more explicit test on this.